### ISSUE TITLE
travis encrypt-file -X:  Argument Error: wrong number of arguements (1 or 2)

### ISSUE THREAD

**Issue by tsugonko** [travis-ci/travis.rb#711]
I'm trying to upload a RSA key for implementation on my project repository, getting the above-mentioned error when using the following cmd:

$ touch .travis.yml && travis encrypt-file ./deploy_rsa --add -r <repositoryName>

$DANGER ZONE: Override existing deploy_rsa.enc? |no| yes 
storing secure env variables for decryption wrong number of arguments (1 for 2)
for a full error report, run travis report -X

Does anybody have any history / guidance with this error?

Thanks,
Stephan

**Comment by ckeyes88** [comment]
I'm also seeing this after updating my travis-cli version to 1.8.11. Have not found a work around yet.

**Comment by ckeyes88** [comment]
Confirmed that down grading to 1.8.8 fixed the issue so there must be something we're missing in the docs or a bug in the CLI for the latest version that's causing this.

**Comment by tsugonko** [comment]
Thanks for the info Chip!!

**Comment by MakerGYT** [comment]
I also encountered this problem
```sh
travis encrypt-file ~/.ssh/id_rsa --add --com
```
`wrong number of arguments (given 1, expected 2)
for a full error report, run travis report --pro`

**Comment by cheshirekow** [comment]
I also hit this problem with 1.8.11. I rolled back to 1.8.10 and it works fine. So the problem appears to be between these two versions.

**Comment by InnocentRoot** [comment]
Thanks !  It worked

**Comment by ind1go** [comment]
It looks like this commit may be the problematic one: https://github.com/travis-ci/travis.rb/commit/cb45412de8144d90bcc88c0525ff8d06d195f512

**Comment by BanzaiMan** [comment]
Should be resolved by #715. Please test it with:

```
gem install travis --pre
```

**Comment by Ahaochan** [comment]
I have never used ruby and gem.
I don't know how to rollback version.

After search, I use this command to solve it.
Hope to help you.
```
gem install travis --version '1.8.10'
```

**Comment by BanzaiMan** [comment]
Sorry for the delay. I'm planning to release a new version with this fix some time next week.

**Comment by BanzaiMan** [comment]
We released 1.8.12 earlier this week.
